### PR TITLE
allow user supplied qr code function

### PR DESCRIPTION
**Pull request by SamHellawell**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -88,6 +87,9 @@ function getIssuerName({ issuer }, didMap) {
 }
 
 function getLikelyImage(s) {
```
Comment (mike-parkhill) [inline src/index.js:89]: how did I let a variable called `s` get through before?

File: src/index.js [hunk 2]
```diff
@@ -225,7 +224,7 @@ export async function getVCData(credential, options = {}) {
   const formatter = new Intl.DateTimeFormat('en-US', { year: 'numeric', month: 'long', day: 'numeric' });
   const date = formatter.format(issuanceDate);
   const expirationDateStr = expirationDate ? formatter.format(expirationDate) : '';
-  const qrImage = generateQR && (await generateQRImage(credential, qrUrl));
+  const qrImage = generateQRImage && (await generateQRImage(credential));
```
Comment (mike-parkhill) [inline src/index.js:227]: should we validate that `generateQRImage` is actually a function before trying to invoke it?

Comment (SamHellawell) [reply src/index.js:227]: yeah guess we can, theres quite a bit of cleanup this lib needs

File: src/index.js (hunk 1 above)
Comment (SamHellawell) [reply src/index.js:89]: not sure, probably from dodeepsearch

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**